cmd/influxd/run.go

```diff
@@ -128,35 +106,92 @@ func parseConfig(path, hostname string) *Config {
 	return config
 }
 
-// creates and initializes a broker at a given path.
-func openBroker(path, addr string) *messaging.Broker {
+// creates and initializes a broker.
+func openBroker(path string, u *url.URL, initializing bool, joinURLs []*url.URL) *messaging.Broker {
+	// Ignore if there's no existing broker and we're not initializing or joining.
+	if !fileExists(path) && !initializing && len(joinURLs) == 0 {
```

Comment:
I had it in there originally but pull it out. I can't remember why at this point tho.